Descent3/mission_download.cpp

```diff
@@ -694,7 +693,7 @@ void _get_zipfilename(char *output, char *directory, char *zipfilename) {
 // return 0 on failure
 // return 1 on success
 int msn_ExtractZipFile(char *zipfilename, char *mn3name) {
-
+/*
```

Comment:
so you just disable the feature ?

Reply:
Most of "mission download" was already commented out due to lack of a location to download from. The only thing I did was comment out the unzipping after it was downloaded. This seems to be the only place that the deflate decompression algorithm is used.

Reply:
Mission files hosted by a server contain a url pointing clients to the download location. If zlib is meant to replace, it should be integrated here.

Reply:
Presently, downloading is entirely disabled. Replacing the compression code right now is a moot point and cannot be tested.

Reply:
This change will make our lives harder when/if we want to enable mission download again, I don't think we should merge it as is